glean_parser/templates/go_server.jinja2

```diff
@@ -0,0 +1,263 @@
+{# The final Go code is autogenerated, but this template is not. Please file bugs! #}
+package glean
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// AUTOGENERATED BY glean_parser v{{ parser_version }}. DO NOT EDIT.
+
+// required imports
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// log type string used to identify logs to process in the Moz Data Pipeline
+var gleanEventMozlogType string = "glean-server-event"
+
+// interface to send the log messages, accepting a call passing in the eventType and the message.
+// TODO: confirm what calls to cloud logging would be required to see if the log method signature makes sense
+// TODO: confirm if this should depend on https://github.com/mozilla-services/go-mozlog
+type Logger interface {
+	log(eventType string, message string)
+}
+
+type GleanEventsLogger struct {
+	AppID             string
+	AppDisplayVersion string
+	AppChannel        string
+	Logger            Logger
+}
+
+// exported type for public method parameters
+type RequestInfo struct {
+	UserAgent string
+	IpAddress string
+}
+
+// TODO: verify with data team what defaults make sense for user agent / ip address if they are not supplied
+var defaultRequestInfo = RequestInfo{
+	UserAgent: "Unknown",
+	IpAddress: "Unknown",
+}
```

Comment:
We should omit these from the logged message.

For context: we are sending some `Unknown` values because these fields are [required by the Glean schema](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/8004ddad52e2c27b0381afa383cbf0779e465d7c/schemas/glean/glean/glean.1.schema.json#L80-L88).